### DISCUSSION TITLE
Submission error

### DISCUSSION THREAD

**Discussion by zhi-yi-huang** [PaddlePaddle/PaddleSpatial#101] [BaiduKDDCup2022] [answered]
Dear Organizer,
I got an error: “[Errno 2] No such file or directory: '/home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_turb_location.CSV'” when I submit my code. Is that means there is no location data about turbines in evaluation environment?

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

can you try /home/aistudio/eval_prog/kddcup22-sdwpf-evaluation/data/sdwpf_baidukddcup2022_turb_location.csv, and see whether there still such problem (just change .CSV to .csv).

Thanks.
